src/js/common/schemaform/FormApp.jsx

```diff
@@ -102,6 +124,27 @@ class FormApp extends React.Component {
     return message;
   }
 
+  redirectOrLoad(props) {
+    const firstPagePath = props.routes[props.routes.length - 1].pageList[0].path;
+
+    // If we're logged in and have a saved / pre-filled form, load that
+    if (props.isLoggedIn) {
+      const currentForm = props.formConfig.formId;
+      const isPrefill = props.prefillsAvailable.includes(currentForm);
+      if (props.savedForms.includes(currentForm) || isPrefill) {
```

Comment:
@jbalboni Yeah, I dunno. I can see it go either way. I'm good with whatever, but to be thorough, I'd like to wrangle in UX for this.

@stacyhlinn The way this works right now is if a logged-in user has a pre-filled form but not a saved form, this will load their pre-filled form and put them on page 1 (after the intro). The alternative is landing them on the intro page and having them explicitly load their pre-filled form.

Which way should we go?